### DISCUSSION TITLE
Is there a way to set space as the leader key?

### DISCUSSION THREAD

**Discussion by Archie-2021** [ahrm/sioyek#737] [Q&A]
I just found out about this great PDF reader and trying to set up my config file, I like to set it up very vim-like , but I can't set up `<space>` as my leader key. Sioyek apparently supports keycords but don't know why it is not working. Also how I can unmap keys?

**Comment by ahrm** [comment]
> Also how I can unmap keys?

You can remove the key definition from `prefs.config`.

**Reply by Archie-2021** [reply]
> > Also how I can unmap keys?
> 
> You can remove the key definition from `prefs.config`.

Yeah , that is way. But I think it is not recommended to mess with original config file. Even you mention it in the tutorial video.

Btw, thanks for this wonderful program 31

**Reply by ahrm** [reply]
The only downside is that you have to re-do it when you uprgade sioyek. Also I think there is a `noop` command so you can effectively unmap by adding
```
<space> noop
```
I don't remember if `noop` was included in version 2.0 though.

**Reply by Archie-2021** [reply]
> The only downside is that you have to re-do it when you uprgade sioyek. Also I think there is a `noop` command so you can effectively unmap by adding
> 
> ```
> <space> noop
> ```
> 
> I don't remember if `noop` was included in version 2.0 though.

Thanks , it is not very vital tbh, I configured the config file till the end and have some spare keys to use still, but it is a good option to have not only for a leader key.
Also I just noticed that `sioyek -v` not working for some reason. `sioyek --version` works tho. Installed from `aur/sioyek`.
